pkgs/os-specific/linux/systemd/default.nix

```diff
@@ -36,10 +36,10 @@ in stdenv.mkDerivation {
   # When updating, use https://github.com/systemd/systemd-stable tree, not the development one!
   # Also fresh patches should be cherry-picked from that tree to our current one.
   src = fetchFromGitHub {
-    owner = "NixOS";
+    owner = "flokli";
```

Comment:
I'll push this to `NixOS/systemd` (and change back the expression) once everybody agrees with the changes there.

Reply:
The commit should be reachable through the NixOS repo as well as long as it exists somewhere in the fork network, so this shouldn't even be necessary. (Here this feature is convenient, but it can also be bad, see e.g. https://github.com/nixos/nixpkgs/commit/7b4c9e9d012)

Reply:
Yes, I know. Still, I want to push it properly.